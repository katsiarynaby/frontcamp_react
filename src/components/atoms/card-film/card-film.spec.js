import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import CardFilm from './index';
import { movie } from './test.data';

describe('CardFilm', () => {
    it("CardFilm should render correctly", () => {
        const wrapper = shallow(<CardFilm
            onClick={jest.fn()}
            movie={movie}
        />);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});


