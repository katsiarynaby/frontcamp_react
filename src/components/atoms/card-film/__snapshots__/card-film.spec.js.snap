// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFilm CardFilm should render correctly 1`] = `
<div
  className="card-film"
  onClick={[MockFunction]}
>
  <img
    className="card-film__image"
    src="https://image.tmdb.org/t/p/w500/432BowXw7a4fWXSONxBaFKqvW4f.jpg"
  />
  <div
    className="card-film__release-and-title"
  >
    <div
      className="card-film__title"
    >
      Transformers 7
    </div>
    <div
      className="card-film__date"
    >
      2019
    </div>
  </div>
  <div
    className="card-film__genre"
  >
    Science Fiction&Action&Adventure
  </div>
</div>
`;
